script.js

```diff
@@ -20,8 +27,51 @@ fetch('https://flipsum-ipsum.net/api/icw/v1/generate?ipsum=recipe-ipsum-text-gen
     })
     stringForHtml += '</p>'
     textToTypeContainer.innerHTML += stringForHtml
+    let upcomingWord = document.getElementById(0)
+    upcomingWord.style.color = ("#cc7a00")
```

Comment:
dont need brackets, and this isnt very readable, I would suggest putting the colours into descriptive constants at the top, that will make this make more sense

Reply:
Done